perses/tests/test_repex.py

```diff
@@ -81,10 +80,9 @@ def test_RESTCapableHybridTopologyFactory_repex_neutral_mutation():
             f_ij, df_ij = analyzer.get_free_energy()
             data[f"{wt_name}-{mutant_name}"] = {'free_energy': f_ij[0, -1], 'error': df_ij[0, -1]}
 
-        forward_data = data['ala-thr']['free_energy']
-        reverse_data = data['thr-ala']['free_energy'] * -1
-        assert np.isclose([forward_data], [reverse_data], atol=ENERGY_DIFF_TOLERANCE_KT), \
-            f"ALA-THR is {forward_data}. THR-ALA is {reverse_data}."
+        DDG = abs(data['ala-thr']['free_energy'] - data['thr-ala']['free_energy'] * -1)
+        dDDG = np.sqrt(data['ala-thr']['error'] ** 2 + data['thr-ala']['error'] ** 2)
+        assert DDG < 6 * dDDG, f"DDG ({DDG}) is greater than 6 * dDDG ({6  * dDDG})"
 
 
 @pytest.mark.gpu_needed
```

Comment:
We can skip this test by making the following change.
```suggestion
@pytest.mark.skip(reason="Currently taking too long in CI.")
@pytest.mark.gpu_needed
```